File: bcube-frontend/src/app/models/responses/BookingResponse.ts
import { BookingStatus } from "../BookingStatus";
import { studio } from "../studio";
import { User } from "../user";

export interface BookingResponse {
    id: number,
    user: User,
    studio: studio,
    date: string,
    startTime: string,
    endTime: string,
    status: BookingStatus
}

File: bcube-services/booking-service/src/main/java/com/bcube/bookingservice/service/dto/response/BookingResponse.java
package com.bcube.bookingservice.service.dto.response;

import com.bcube.bookingservice.persistance.entity.BookingStatus;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.Instant;
import java.time.LocalDate;

@Data
@AllArgsConstructor
public class BookingResponse {
    private Long id;
    private Long userId;
    private Long studioId;
    private LocalDate date;
    private Instant startTime;
    private Instant endTime;
    private BookingStatus status;
}